lib/rules-generator.js

```diff
@@ -332,7 +380,11 @@ util.methods(Generator, {
     switch (exp.type) {
```

Comment:
lots of return exp at the end of every case. Can we just have one return exp at the very bottom